src/OpenLoco/src/Input/Shortcuts.cpp

```diff
@@ -582,4 +468,55 @@ namespace OpenLoco::Input::ShortcutManager
     {
         GameCommands::doCommand(GameCommands::SetGameSpeedArgs{ GameSpeed::ExtraFastForward }, GameCommands::Flags::apply);
     }
+
+    void initialize()
+    {
+        ShortcutManager::add(Shortcut::closeTopmostWindow, StringIds::shortcut_close_topmost_window, closeTopmostWindow, "closeTopmostWindow", "Backspace");
```

Comment:
could you play with the formatting a bit here

Reply:
Not exactly sure what I should do, do you want them aligned or put arguments on separate lines?

Reply:
Could you align things. The previous table was easier to read

Reply:
> Could you align things. The previous table was easier to read

I'm inclined to agree, but then I authored the previous table :-)